libs/sdk-ui-dashboard/src/model/store/user/userSelectors.ts

```diff
@@ -12,7 +12,8 @@ const selectSelf = createSelector(
  * This selector returns current logged in user. It is expected that the
  * selector is called only after the permission state is correctly initialized. Invocations before initialization lead to invariant errors.
  *
- * @alpha
+ * @returns - an {@link @gooddata/sdk-model#IUser} object for logged in user.
+ * @public
  */
 export const selectUser = createSelector(selectSelf, (state) => {
```

Comment:
I am a bit concerned about the naming. Should we rename this to `selectCurrentUser` ? To keep this consistent with the other one?